packages/good-design/src/core/layout/ClaimCarousel.tsx

```diff
@@ -44,14 +44,19 @@ const ClaimCarousel: FC<ClaimCarouselProps> = ({ cards }) => {
         horizontal
         onScroll={onScroll}
         scrollEventThrottle={16}
+        mt={8}
+        contentContainerStyle={{
+          flex: 1,
```

Comment:
@alt-a1r 
this will disable scrolling

Reply:
was it solved @roma-hladilka @alt-a1r  ?